### ISSUE TITLE
Error when create Image Classtifier type on Matter

### ISSUE THREAD

**Issue by kojca** [jjdejong/phpip#36]
I cannot add Image Classtifier on Matter (Chrome browser). Because File Input is not showing. I fixed it on app.blade.php as below:

```
if(e.target.previousElementSibling != null){
if (e.target.previousElementSibling.name === 'type_code' && e.target.previousElementSibling.value === 'IMG') {
        for (elt of addClassifierForm.getElementsByClassName('hideForFile')) {
          elt.classList.add('d-none');
        }
        forFile.classList.remove('d-none');
        cTypeCode = 'IMG'
      }
      if (e.target.previousElementSibling.name === 'type_code' && e.target.previousElementSibling.value !== 'IMG' && cTypeCode === 'IMG') {
        for (elt of addClassifierForm.getElementsByClassName('hideForFile')) {
          elt.classList.remove('d-none');
        }
        forFile.classList.add('d-none');
        cTypeCode = ''
      }
   }
```

It right?
Thank you

**Comment by jjdejong** [comment]
Does that also work in FireFox?

**Comment by jjdejong** [comment]
The functionality was also broken in FireFox. I fixed it in 5efd300, although I used a different approach (the "previous sibling" approach would break if the hidden input field is moved elsewhere)
